lib/libmbedtls/mbedtls/library/rsa.c

```diff
@@ -211,9 +211,17 @@ int mbedtls_rsa_check_privkey( const mbedtls_rsa_context *ctx )
     if( ( ret = mbedtls_rsa_check_pubkey( ctx ) ) != 0 )
         return( ret );
 
-    if( !ctx->P.p || !ctx->Q.p || !ctx->D.p )
+    if( !ctx->D.p )
         return( MBEDTLS_ERR_RSA_KEY_CHECK_FAILED );
 
+#if defined(MBEDTLS_RSA_NO_CRT)
+    ((void) PQ); ((void) DE); ((void) P1); ((void) Q1); ((void) H); ((void) I); ((void) G);
+    ((void) G2); ((void) L1); ((void) L2);  ((void) DP); ((void) DQ); ((void) QP);
```

Comment:
why these ?

Reply:
They are used to skip the unused build errors.